website/eslint.config.mjs

```diff
@@ -0,0 +1,50 @@
+import globals from "globals";
+import pluginJs from "@eslint/js";
+import * as tseslint from "typescript-eslint";
+import pluginReactConfig from "eslint-plugin-react/configs/recommended.js";
+import prettierEslintPlugin from "eslint-plugin-prettier";
+import docusaurusEslintPlugin from "@docusaurus/eslint-plugin";
+
+export default [
+  // TODO: eslint configure for .md, .mdx files
+  {files: ["*/.{js,mjs,cjs,ts,jsx,tsx}"]},
+  {ignores: ["babel.config.js", "eslint.config.mjs", "docusaurus.config.ts", ".docusaurus/*", "build/*"]},
+  {
+    plugins: {
+      prettier: prettierEslintPlugin,
+      "@docusaurus": docusaurusEslintPlugin,
+    },
+  },
+  {
+    languageOptions: {
+      parser: tseslint.parser, 
+      parserOptions: {
+        ecmaFeatures: {jsx: true},
+        sourceType: "module"
+      },
+      globals: {
+        ...globals.browser,
+        ...globals.es2021,
+      },
+    },
+  },
+  pluginJs.configs.recommended,
+  ...tseslint.configs.recommended,
+  pluginReactConfig,
+  {
+    settings: {
+      react: {
+        version: "detect",
+      },
+    },
+  },
+  {
+    rules: {
+      "prettier/prettier": "off",
```

Comment:
If I understand correctly, this turns off prettier eslint rules. If that is the case, I'm not sure why the prettier plugin is needed at all.

Reply:
Actually, I was testing for unusual logs while running `npm run lint`, and it was giving some 1000+ logs, so I had disabled it temporarily for debugging purposes. Forgot to change it to `warn`. Updated in the latest commit.